_typos.toml

```diff
@@ -0,0 +1,14 @@
+[default]
+extend-ignore-identifiers-re = [
+    # Related to serialized object
+    "ser.*",
```

Comment:
ser_.*

maybe ?

to avoid matching more stuff than necessary

Reply:
Since we are in the section `extend-ignore-identifiers-re` we can simply put  `ser` to ignore case like `ser =` `ser.some_call()` and `pks_ser = ` for example.